website/docs/dbt-cli/install/from-source.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Install from source"
+id: "from-source"
+---
+
+dbt Core is open source software, as are almost all adapter plugins. As such, the codebases are freely available to download and build from source. To do this, you can clone the repositories from GitHub and install the local version using `pip`.
+
+This is also an important part of contributing. If you'd like to help fix a bug or implement a sought-after feature. For more details, read the [core contributing guide](https://github.com/dbt-labs/dbt/blob/HEAD/CONTRIBUTING.md).
+
+### Core
+
+To install from source:
+
+```shell
+git clone https://github.com/dbt-labs/dbt.git
+cd dbt
+pip install -r requirements.txt
+```
+
+This will install `dbt-core` and `dbt-postgres`. To install in editable mode (includes your local changes as you make them), use `pip install -e editable-requirements.txt` instead.
+
+### Plugin
+
+To install a plugin from source:
```

Comment:
Do you need to install core _and_ the adapter, or do adapters bring core in with them? We should clarify either way. The suggestion below assumes you need both.

Since it doesn’t _strictly_ need to be before, I'd be OK with "in addition to installing your plugin", but in practice everyone will do core before the adapter I suspect.

```suggestion

:::info

You need to install dbt-core as described above before installing your plugin. 

:::

To install a plugin from source:
```